File: golang/HelloWorld.go
package main

import "fmt"

func main() {
    fmt.Println("HelloWorld")
    a:=10
    b:=20 
    c:=plus(a,b)
    fmt.Printf("plus=%d\n",c)
    d:=number{a,b}
    e:=d.minus()
    fmt.Printf("minus=%d\n",e)
}

func plus(a int, b int) int {
    return a+b
}

type number struct {
    a int
    b int
}

func (n number) minus() int {
    return n.a-n.b
}

File: python/HelloWorld.py
def plus(a,b):
    return a+b

class Sample:
    num = 0
    def __init__(self):
        self.num+=1

    def minus(self,a,b):
        return a-b

    def display(self):
        print "{0}{1}".format("num=",self.num)
